Committing changes for signInFrequency updates

rckyplln opened the pull request.

opbld34 commented:
Docs Build status updates of commit _[1911301](https://github.com/rckyplln/microsoft-graph-docs/commits/1911301b2dadd4c543ae5259b164efa8f96848ba)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/enums.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[api-reference/beta/resources/signinfrequencysessioncontrol.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/signinfrequencysessioncontrol.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/signinfrequencysessioncontrol?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C8%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203080506343472-16097%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-08T05%3A08%3A37Z&sr=b&sp=r&sig=RKPGUuX2Lcyrtm9phGOFCy0ksqLU1Tvezk5OQB4pesU%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

Lauragra commented:
@rckyplln , we will need a changelog entry for the new properties/enum types. Also requesting that @FaithOmbongi review and approve. Thanks.

opbld31 commented:
Docs Build status updates of commit _[6dcaa86](https://github.com/rckyplln/microsoft-graph-docs/commits/6dcaa864b4af0f01ac4c232e64093042a4aaa6bb)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/enums.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[api-reference/beta/resources/signinfrequencysessioncontrol.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/signinfrequencysessioncontrol.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/signinfrequencysessioncontrol?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C14%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203142105020256-16097%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-14T21%3A07%3A13Z&sr=b&sp=r&sig=HP2bO%2BKxYDRCtEF%2B%2B%2FqfJxhO%2FHSWiasTTlCm3qP2mMk%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

rckyplln commented:
Hi all,

Apologies for the delay, here’s the changelog information: Pull request 5795065: Public preview reauth - Repos (visualstudio.com)<https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5795065?_a=files&path=%2FChangelog%2F5795065-Public-preview-reauth%2FMicrosoft.IdentityProtectionServices.json>
I had to make one last update today and put the isEnabled attribute back. The latest commit is there.

Let me know what else is needed. I apologize for all of the back and forth—this process is all new to me! 😅

/ricky

From: Laura Graham ***@***.***>
Sent: Tuesday, March 8, 2022 7:14 PM
To: microsoftgraph/microsoft-graph-docs ***@***.***>
Cc: Ricky Pullan ***@***.***>; Mention ***@***.***>
Subject: Re: [microsoftgraph/microsoft-graph-docs] Committing changes for signInFrequency updates (PR #16097)


@rckyplln<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Frckyplln&data=04%7C01%7Cricky.pullan%40microsoft.com%7Cb732f3ff087444f6520608da017aeac6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637823924696952478%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=%2BM5ahEcNpd7TUeE1N0GTtBdQg23JRb7GaHHUfEN14qg%3D&reserved=0> , we will need a changelog entry for the new properties/enum types. Also requesting that @FaithOmbongi<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FFaithOmbongi&data=04%7C01%7Cricky.pullan%40microsoft.com%7Cb732f3ff087444f6520608da017aeac6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637823924696952478%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=qdRD3XT41YsdKj89P6LpQKUvW%2By%2FgqoL13cY2ePfcKY%3D&reserved=0> review and approve. Thanks.

—
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoftgraph%2Fmicrosoft-graph-docs%2Fpull%2F16097%23issuecomment-1062512134&data=04%7C01%7Cricky.pullan%40microsoft.com%7Cb732f3ff087444f6520608da017aeac6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637823924696952478%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=fuFKwyvVznM8osThQprzFZMNUi3l9gO9HYmjHktLt1I%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAWFFZ2X57VADOW7S6MN56LTU7AJRFANCNFSM5QFI7EFQ&data=04%7C01%7Cricky.pullan%40microsoft.com%7Cb732f3ff087444f6520608da017aeac6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637823924696952478%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=1dg4oeJ9wbKVFHDjsmXlDKYZlIT7LnlD8MJbqnrRf2g%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cricky.pullan%40microsoft.com%7Cb732f3ff087444f6520608da017aeac6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637823924696952478%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=H%2BhUWPH8erWroysKbUtsx2IknDRkh1NAoUdyol6Jny4%3D&reserved=0> or Android<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cricky.pullan%40microsoft.com%7Cb732f3ff087444f6520608da017aeac6%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637823924697002459%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=Q3SOW9OYXNasYcCw%2FwMzC0tnaIG8EHHSjh4Z%2Ft7w9vM%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

rckyplln commented:
Here is the changelog, as requested: https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5795065?_a=files&path=/Changelog/5795065-Public-preview-reauth/Microsoft.IdentityProtectionServices.json

Lauragra commented:
@rckyplln , please follow the [Get started](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211776/Get-started-with-GitHub) steps to set yourself up to be a contributor. Then, you can add the changelog entry that was generated directly to this file in your branch: https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/changelog/Microsoft.IdentityProtectionServices.json. When you're ready for us to review your PR, please set the **ready for content review** label. Thanks.

rckyplln commented:
Hi Laura,

Thank you for your guidance here. I went through the getting started guide, and added myself to the various Groups and Security Groups. I added the labels as request below and also created an ADO work item: Product Backlog Item 1821923: [PROPOSED] Sign-in frequency API updates for re-authentication feature release - Boards (visualstudio.com)<https://identitydivision.visualstudio.com/Technical%20Content/_workitems/edit/1821923>.

Thank you again!
/ricky

From: Laura Graham ***@***.***>
Sent: Monday, March 14, 2022 6:46 PM
To: microsoftgraph/microsoft-graph-docs ***@***.***>
Cc: Ricky Pullan ***@***.***>; Mention ***@***.***>
Subject: Re: [microsoftgraph/microsoft-graph-docs] Committing changes for signInFrequency updates (PR #16097)


@rckyplln<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Frckyplln&data=04%7C01%7Cricky.pullan%40microsoft.com%7C5facf5419e7c4f1b7eaa08da06257fac%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637829055385717379%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=yphwm%2BKnSTrT3lb6Jal9n60jWFgfshHG%2BKqTe%2BkyL60%3D&reserved=0> , please follow the Get started<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fdev.azure.com%2Fmsazure%2FOne%2F_wiki%2Fwikis%2FMicrosoft%2520Graph%2520Partners%2F211776%2FGet-started-with-GitHub&data=04%7C01%7Cricky.pullan%40microsoft.com%7C5facf5419e7c4f1b7eaa08da06257fac%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637829055385717379%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=SNQ0hOI2cfaFyqxPnMkv2diwRIcXYxRubf2QD7VyuBU%3D&reserved=0> steps to set yourself up to be a contributor. Then, you can add the changelog entry that was generated directly to this file in your branch: https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/changelog/Microsoft.IdentityProtectionServices.json<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoftgraph%2Fmicrosoft-graph-docs%2Fblob%2Fmain%2Fchangelog%2FMicrosoft.IdentityProtectionServices.json&data=04%7C01%7Cricky.pullan%40microsoft.com%7C5facf5419e7c4f1b7eaa08da06257fac%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637829055385717379%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=hM8WXLcu418A1AxIQMzXA9MSF%2Bt%2BktM95l4ZlAc5rnI%3D&reserved=0>. When you're ready for us to review your PR, please set the ready for content review label. Thanks.

-
Reply to this email directly, view it on GitHub<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fmicrosoftgraph%2Fmicrosoft-graph-docs%2Fpull%2F16097%23issuecomment-1067475169&data=04%7C01%7Cricky.pullan%40microsoft.com%7C5facf5419e7c4f1b7eaa08da06257fac%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637829055385717379%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=jS5X0iDy7SY33cghaYP%2BRp%2F2DEwnvKglzfq0TeabYRg%3D&reserved=0>, or unsubscribe<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAWFFZ2QA5FSWQETMH4SDJSTU77TT5ANCNFSM5QFI7EFQ&data=04%7C01%7Cricky.pullan%40microsoft.com%7C5facf5419e7c4f1b7eaa08da06257fac%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637829055385717379%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=6Y7W9OPq4o9epvE3%2FDNkNEWLn0ZLoWJkX4HRohibaDU%3D&reserved=0>.
Triage notifications on the go with GitHub Mobile for iOS<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fapps.apple.com%2Fapp%2Fapple-store%2Fid1477376905%3Fct%3Dnotification-email%26mt%3D8%26pt%3D524675&data=04%7C01%7Cricky.pullan%40microsoft.com%7C5facf5419e7c4f1b7eaa08da06257fac%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637829055385717379%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=eRLr9Yexs7wmDJQGXOg9GBkvs0aVRhKue05Zj9LLv%2BI%3D&reserved=0> or Android<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fplay.google.com%2Fstore%2Fapps%2Fdetails%3Fid%3Dcom.github.android%26referrer%3Dutm_campaign%253Dnotification-email%2526utm_medium%253Demail%2526utm_source%253Dgithub&data=04%7C01%7Cricky.pullan%40microsoft.com%7C5facf5419e7c4f1b7eaa08da06257fac%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637829055385717379%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=vg7%2FRmXRLReVKC8M7EjzaWL4oZa1OWVSVmdMyLkmVNw%3D&reserved=0>.
You are receiving this because you were mentioned.Message ID: ***@***.******@***.***>>

opbld31 commented:
Docs Build status updates of commit _[254a6ef](https://github.com/rckyplln/microsoft-graph-docs/commits/254a6efc0c22a79d7471abbf94de6a8c34b9deae)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/enums.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[api-reference/beta/resources/signinfrequencysessioncontrol.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/signinfrequencysessioncontrol.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/signinfrequencysessioncontrol?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[changelog/Microsoft.IdentityProtectionServices.json](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/changelog/Microsoft.IdentityProtectionServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C17%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203172111312950-16097%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-17T21%3A14%3A45Z&sr=b&sp=r&sig=MCIhUJiGZV5iNR73K6%2BBiYaVTqgviTwqTMgotxQyNJI%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

rckyplln commented:
#feedback-addressed

FaithOmbongi commented on api-reference/beta/resources/signinfrequencysessioncontrol.md:
```diff
@@ -20,8 +20,10 @@ Session control to enforce sign-in frequency. Inherits from [Conditional Access
-|type          |signinFrequencyType       | Possible values are: `days`, `hours`.|
+|type          |signinFrequencyType       | Possible values are: `days`, `hours`, or `null` if frequencyInterval is everyTime .|
 |value         |Int32        | The number of `days` or `hours`.|
+|authenticationType |signInFrequencyAuthenticationType  | Possible values are `primaryAndSecondaryAuthentication` or `secondaryAuthentication` .|
+|frequencyInterval  |signInFrequencyInterval  | Possible values are `timeBased` or `everyTime` .|
```
`unknownFutureValue` helps customers know these are evolvable enums and they can handle future changes in their code
```suggestion
|authenticationType |signInFrequencyAuthenticationType  | The possible values are `primaryAndSecondaryAuthentication`, `secondaryAuthentication`, `unknownFutureValue`.|
|frequencyInterval  |signInFrequencyInterval  | The possible values are `timeBased`, `everyTime`, `unknownFutureValue`.|
```

FaithOmbongi commented on api-reference/beta/resources/signinfrequencysessioncontrol.md:
````diff
@@ -42,9 +44,10 @@ The following is a JSON representation of the resource.
 
 ```json
 {
-  "isEnabled": true,
````
Please restore the **isEnabled** property, the type will be "Boolean" though - this property is inherited from the **conditionalAccessSessionControl** base type. Unless it's not returned in the response payload - then that's a different conversation.

FaithOmbongi commented on changelog/Microsoft.IdentityProtectionServices.json:
```diff
@@ -1,5 +1,47 @@
 	"changelog": [
+		{
+			"ChangeList": [
+			  {
+				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
+				"ApiChange": "EnumType",
+				"ChangedApiName": "signInFrequencyAuthenticationType",
+				"ChangeType": "Addition",
+				"Description": "Added the [signInFrequencyAuthenticationType](https://docs.microsoft.com/en-us/?view=graph-rest-beta) resource",
+				"Target": "signInFrequencyAuthenticationType"
+			  },
+			  {
+				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
+				"ApiChange": "EnumType",
+				"ChangedApiName": "signInFrequencyInterval",
+				"ChangeType": "Addition",
+				"Description": "Added the [signInFrequencyInterval](https://docs.microsoft.com/en-us/?view=graph-rest-beta) resource",
+				"Target": "signInFrequencyInterval"
+			  },
+			  {
+				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
+				"ApiChange": "Property",
+				"ChangedApiName": "authenticationType",
+				"ChangeType": "Addition",
+				"Description": "Added the [authenticationType](https://docs.microsoft.com/en-us/graph/api/resources/signInFrequencySessionControl?view=graph-rest-beta) resource",
+				"Target": "signInFrequencySessionControl"
+			  },
+			  {
+				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
+				"ApiChange": "Property",
+				"ChangedApiName": "frequencyInterval",
+				"ChangeType": "Addition",
+				"Description": "Added the [frequencyInterval](https://docs.microsoft.com/en-us/graph/api/resources/signInFrequencySessionControl?view=graph-rest-beta) resource",
+				"Target": "signInFrequencySessionControl"
+			  }
```
The descriptions were off. If they were autogenerated like this by Graph Studio, please share in ADO the link to the schema PR so I can file a bug report with the onboarding team.
```suggestion
			  {
				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
				"ApiChange": "Enum type",
				"ChangedApiName": "signInFrequencyAuthenticationType",
				"ChangeType": "Addition",
				"Description": "Added the **signInFrequencyAuthenticationType** enumeration.",
				"Target": "signInFrequencyAuthenticationType"
			  },
			  {
				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
				"ApiChange": "Enum type",
				"ChangedApiName": "signInFrequencyInterval",
				"ChangeType": "Addition",
				"Description": "Added the **signInFrequencyInterval** enumeration.",
				"Target": "signInFrequencyInterval"
			  },
			  {
				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
				"ApiChange": "Property",
				"ChangedApiName": "authenticationType",
				"ChangeType": "Addition",
				"Description": "Added the **authenticationType** property to the [signInFrequencySessionControl](https://docs.microsoft.com/en-us/graph/api/resources/signInFrequencySessionControl?view=graph-rest-beta) resource.",
				"Target": "signInFrequencySessionControl"
			  },
			  {
				"Id": "1ea2aa9b-44d5-43b9-9b47-73ab6b4de86c",
				"ApiChange": "Property",
				"ChangedApiName": "frequencyInterval",
				"ChangeType": "Addition",
				"Description": "Added the **frequencyInterval** property to the [signInFrequencySessionControl](https://docs.microsoft.com/en-us/graph/api/resources/signInFrequencySessionControl?view=graph-rest-beta) resource.",
				"Target": "signInFrequencySessionControl"
			  }
```

FaithOmbongi reviewed (changes requested):
Thank you @rckyplln for your work in this PR. A few fixes and the PR will be good for approval.

opbld32 commented:
Docs Build status updates of commit _[e3ceffd](https://github.com/rckyplln/microsoft-graph-docs/commits/e3ceffd38f0765917de259640900f7e1a2b32eac)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/enums.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[api-reference/beta/resources/signinfrequencysessioncontrol.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/signinfrequencysessioncontrol.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/signinfrequencysessioncontrol?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[changelog/Microsoft.IdentityProtectionServices.json](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/changelog/Microsoft.IdentityProtectionServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203181842064006-16097%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-18T18%3A44%3A17Z&sr=b&sp=r&sig=lMYPfXwO8YN%2FgvL%2FQRA%2FEcoCGxPXVXKrN05vAW0sq2c%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld30 commented:
Docs Build status updates of commit _[ff0fe42](https://github.com/rckyplln/microsoft-graph-docs/commits/ff0fe426bf9717d4a416998f1c132c88f1739a30)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-314fc4f6)

<a id="314fc4f6"></a>
###
 - **[Error: PullRequestMergeConflict]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203181842375347-16097%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-18T18%3A44%3A47Z&sr=b&sp=r&sig=lCG3rppLrs%2BhyuIg8Q0d2vaPaorcy2p%2BsoxZrtZLj%2F4%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld32 commented:
Docs Build status updates of commit _[4290ae9](https://github.com/rckyplln/microsoft-graph-docs/commits/4290ae9bfcc69f2674771f65f45c7cc185b85e23)_: 

### :x: Validation status: errors

Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main#build--publish) which may help to resolve issue.


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[]() | :x:Error | | [Details](#user-content-f16b8c49)

<a id="f16b8c49"></a>
###
 - **[Error: PullRequestMergeConflict]** `````Cannot sync git repo to specified state. Please make sure the pull request is mergeable.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203181849264373-16097%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-18T18%3A50%3A16Z&sr=b&sp=r&sig=qSbpl3WDGaKhuYEU8JXDbFUsJljJl51uGwZPrysFjYE%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

rckyplln commented:
Was getting a merge error-- appears to be fixed. Ready for review once more. Thanks for your help and patience on this!

opbld30 commented:
Docs Build status updates of commit _[e15c0f4](https://github.com/rckyplln/microsoft-graph-docs/commits/e15c0f40a27c412a985c55a46f33dbea96c67131)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/enums.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[api-reference/beta/resources/signinfrequencysessioncontrol.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/signinfrequencysessioncontrol.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/signinfrequencysessioncontrol?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[changelog/Microsoft.IdentityProtectionServices.json](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/changelog/Microsoft.IdentityProtectionServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203181903313889-16097%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-18T19%3A05%3A40Z&sr=b&sp=r&sig=RKXdNd%2BaF7Hkns3OJfTk0C0zcQh0Y0ixqW8OfkEQ3gc%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

opbld30 commented:
Docs Build status updates of commit _[2de533c](https://github.com/rckyplln/microsoft-graph-docs/commits/2de533ca28644316f6f6c6b9808fb579c9c41d46)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/enums.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[api-reference/beta/resources/signinfrequencysessioncontrol.md](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/api-reference/beta/resources/signinfrequencysessioncontrol.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/signinfrequencysessioncontrol?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[changelog/Microsoft.IdentityProtectionServices.json](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/changelog/Microsoft.IdentityProtectionServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C3%5C18%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202203181911432051-16097%5Cworkflow_report.html?sv=2020-08-04&se=2022-04-18T19%3A14%3A08Z&sr=b&sp=r&sig=L1EDLu7HcODvsaa9fEIcZsOsxV70jXtinxHREbpARWc%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

rckyplln commented:
@FaithOmbongi , not sure what the above means-- is any action required from me?

FaithOmbongi commented:
Hi @rckyplln, you'll need to resolve the merge conflict to get your PR merged when you're ready to merge it.

opbld30 commented:
Docs Build status updates of commit _[5077cae](https://github.com/rckyplln/microsoft-graph-docs/commits/5077cae0e924fe557e0393714c883e20a5fa032a)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/beta/resources/enums.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/enums.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/enums?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[api-reference/beta/resources/signinfrequencysessioncontrol.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/resources/signinfrequencysessioncontrol.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/signinfrequencysessioncontrol?view=graph-rest-beta&branch=pr-en-us-16097) (graph-rest-beta) |
[changelog/Microsoft.IdentityProtectionServices.json](https://github.com/rckyplln/microsoft-graph-docs/blob/2022-Mar-7_Sign-InFrequencyUpdates/changelog/Microsoft.IdentityProtectionServices.json) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://buildapi.docs.microsoft.com/Output/PullRequest/77e68b91-cf14-3558-43ca-0d6d735c492d/202206080245127528-16097/BuildReport?accessString=3e53a0d6e46e2f42ca1e156e6a5d2e9cc63e46a889dac9b078c99a9076a4f49f).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>
